pixman/src/image/bits.rs

```diff
@@ -467,7 +471,9 @@ impl<'bits, 'alpha> Image<'bits, 'alpha> {
     ///
     /// # Safety
     ///
-    /// The pointer is expected to be valid and have a ref-count of at least one.
+    /// The pointer is expected to be valid and have a ref-count of at *exactly* one,
+    /// with an alpha map (if any) that has a ref count of one.
+    ///
```

Comment:
Perhaps also adding a comment that increasing the ref-count by by-passing pixman-rs is breaking `Send`?